### PR TITLE
Malware - Adding MDE playbooks

### PR DESCRIPTION
**Pull request by ssokolovich**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Microsoft Defender PBs for the malware use-case 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [x] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by julieschwartz18** [comment]
Doc review complete for the playbook-MDE_-_False_Positive_Incident_handling.yml file. @ssokolovich - please review the following and confirm:
- updates to descriptions throughout - please also regenerate the README file.
- changed MSDE to MDE (also in task names)
- added the word Cortex to XSOAR (also in task names)
- in one place, changed demisto to Cortex XSOAR

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Host_Advanced_Hunting.yml [hunk 1]
```diff
@@ -0,0 +1,1007 @@
...
+      - '38'
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ''
+      id: 2a614f24-f7ad-4297-8d4a-42cceb520fe3
+      iscommand: false
+      name: Mimikatz/Other Lsaas manipulation
```
Comment (julieschwartz18) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Host_Advanced_Hunting.yml:218]: Just to confirm - is this task name ok?

**Comment by julieschwartz18** [comment]
@ssokolovich - reviewed all the yaml files. There are updates to the descriptions throughout, please regenerate the README.md files. Thanks

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_False_Positive_Incident_Handling.yml [hunk 2]
```diff
@@ -0,0 +1,831 @@
...
+        "height": 1950,
+        "width": 1570,
+        "x": -410,
+        "y": -220
+      }
+    }
+  }
+tests:
+  - No tests
+fromversion: 6.2.0
```
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_False_Positive_Incident_Handling.yml:831]: 6.5

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Host_Advanced_Hunting.yml [hunk 3]
```diff
@@ -0,0 +1,1007 @@
...
+        "height": 1135,
+        "width": 3300,
+        "x": -1920,
+        "y": 60
+      }
+    }
+  }
+tests:
+- Microsoft Defender Advanced Threat Protection - Test
+fromversion: 6.2.0
```
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Host_Advanced_Hunting.yml:1007]: 6.5

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Host_Advanced_Hunting_For_Network_Activity.yml [hunk 4]
```diff
@@ -0,0 +1,1548 @@
...
+        "height": 2005,
+        "width": 3440,
+        "x": 1020,
+        "y": 410
+      }
+    }
+  }
+tests:
+- No tests (auto formatted)
+fromversion: 6.2.0
```
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Host_Advanced_Hunting_For_Network_Activity.yml:1548]: 6.5

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Host_Advanced_Hunting_README.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
+This playbook will use the Microsoft Defender For Endpoint feature - Advanced Hunting. The hunt will be done based on provided inputs.
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+* 1da35988-954b-489f-8973-89a62b0cc89c
```
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Host_Advanced_Hunting_README.md:7]: id

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Pro-Active_Actions.yml [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+  playbookInputQuery:
+- key: machine_ID
+  value: {}
+  required: false
+  description: Provide the machine IDs of the systems you want to retrieve.
+  playbookInputQuery: null
+outputs: []
+tests:
+- No tests (auto formatted)
+fromversion: 6.0.0
```
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Pro-Active_Actions.yml:269]: 6.5.0

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Pro-Active_Actions_README.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+- Run Full AV scan for spesific enpoint
+- Request Investigation Package (zip file that contains forensic data - with a size of ~ 15MB) from an endpoint.
+- Request to run Automatic Investigation on an endpoint.
+
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+* 8d6603be-227b-48fb-8f77-0b0ff3d572db
```
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Pro-Active_Actions_README.md:11]: id

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Retrieve_File.yml [hunk 8]
```diff
@@ -0,0 +1,263 @@
...
+        "height": 925,
+        "width": 842.5,
+        "x": -300,
+        "y": -140
+      }
+    }
+  }
+tests:
+- No tests (auto formatted)
+fromversion: 6.2.0
```
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Retrieve_File.yml:263]: 6.5.0

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Retrieve_File_README.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+* isError
+* UnzipFile
+
+### Commands
+* microsoft-atp-live-response-get-file
+
+## Playbook Inputs
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
```
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Retrieve_File_README.md:22]: No input for host ID?

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_Malware_-_Incident_Enrichment_README.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
+This playbook aims to enrich Microsoft Defender For Endpoint alert. The enrichment is done on: the involved endpoint, Mitre technique ID information, and sets the 'Malware-Investigation and Response' layout.
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+* Mitre Attack - Extract Technique Information From ID
+* 1d1d0c8b-f429-4cbb-8de5-a0fb2742f1d6
```
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_Malware_-_Incident_Enrichment_README.md:8]: id

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_Malware_-_Investigation_and_Response_README.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+- Sandbox hash search and detonation
+- Pro-Active investigation actions: Av scan, Investigation Package collection, Run Automated Investigation on an endpoint
+- Microsoft Defender For Endpoint alert enrichment
+- Incident Handling ( True/False Positive)
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+* 470ceb33-1487-47a0-8337-a6c50149f385
```
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_Malware_-_Investigation_and_Response_README.md:14]: ID

File: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_11_0.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+#### Integrations
+- **Microsoft Defender for Endpoint**
+- Set Fetch incident to be true by default.
+- Set Default mapper to be **Microsoft Defender For Endpoint Mapper**
+
+#### Mappers
+- **Microsoft Defender For Endpoint Mapper**
+- Added new fields to be associated with new malware type
+
+#### Playbooks
```
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_11_0.md:11]: all 6.2 to 6.5

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_SIEM_ingestion_-_Get_Incident_Data.yml [hunk 13]
```diff
@@ -0,0 +1,426 @@
...
+        "height": 865,
+        "width": 630,
+        "x": 230,
+        "y": 80
+      }
+    }
+  }
+tests:
+- No tests (auto formatted)
+fromversion: 6.2.0
```
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_SIEM_ingestion_-_Get_Incident_Data.yml:426]: 6.5

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_Malware_-_Investigation_and_Response.yml [hunk 14]
```diff
@@ -0,0 +1,1968 @@
...
+        "height": 3935,
+        "width": 2340,
+        "x": -750,
+        "y": -80
+      }
+    }
+  }
+tests:
+- No tests (auto formatted)
+fromversion: 6.2.0
```
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_Malware_-_Investigation_and_Response.yml:1968]: 6.5

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_Malware_-_Incident_Enrichment.yml [hunk 15]
```diff
@@ -0,0 +1,1164 @@
...
+        "height": 2025,
+        "width": 1520,
+        "x": -230,
+        "y": 370
+      }
+    }
+  }
+tests:
+- No tests (auto formatted)
+fromversion: 6.2.0
```
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_Malware_-_Incident_Enrichment.yml:1164]: 6.5

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_True_Positive_Incident_Handling.yml [hunk 16]
```diff
@@ -0,0 +1,1404 @@
...
+        "width": 2530,
+        "x": -2780,
+        "y": -170
+      }
+    }
+  }
+tests:
+- Microsoft Defender Advanced Threat Protection - Test
+- Microsoft Defender - ATP - Indicators SC Test
+fromversion: 6.2.0
```
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_True_Positive_Incident_Handling.yml:1404]: 6.5

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Host_Advanced_Hunting_For_Powershell_Executions.yml [hunk 17]
```diff
@@ -0,0 +1,773 @@
...
+        "height": 1925,
+        "width": 1700,
+        "x": -1290,
+        "y": 60
+      }
+    }
+  }
+tests:
+- No tests (auto formatted)
+fromversion: 6.2.0
```
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Host_Advanced_Hunting_For_Powershell_Executions.yml:773]: 6.5

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Host_Advanced_Hunting_For_Persistence.yml [hunk 18]
```diff
@@ -0,0 +1,1165 @@
...
+        "height": 895,
+        "width": 3000,
+        "x": -1470,
+        "y": 410
+      }
+    }
+  }
+tests:
+- No tests (auto formatted)
+fromversion: 6.2.0
```
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Host_Advanced_Hunting_For_Persistence.yml:1165]: 6.5

**Comment by julieschwartz18** [comment]
Doc review complete for the playbook-MDE_-_Host_Advanced_Hunting_README.md
 file, aligned the descriptions with the yaml. @ssokolovich - the sub playbook names need to be updated in the yaml (they currently show the id)

**Comment by julieschwartz18** [comment]
Doc review complete for the playbook-MDE_-_Pro-Active_Actions_README.md file, aligned descriptions with the yaml.
@ssokolovich - the sub playbook name and playbookId need to be updated in the yaml

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Retrieve_File_README.md (hunk 9 above)
Comment (julieschwartz18) [reply Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Retrieve_File_README.md:22]: I didn't see an input for host ID in the yaml

**Comment by julieschwartz18** [comment]
Doc review complete for this file. @ssokolovich - please add the sub playbook name in line 8, I wasn't able to figure it out from the yaml. The yaml also needs to be updated with the sub playbook name and playbookId

**Comment by julieschwartz18** [comment]
Doc review complete for the playbook-MDE_Malware_-_Investigation_and_Response_README.md file. @ssokolovich -
- Missing sub playbook names (and playbookIds) in the yaml
- The input table is missing the **DidAlertOriginateFromSIEM** input
- Please regenerate this file from the yaml

File: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_11_0.md (hunk 12 above)
Comment (julieschwartz18) [reply Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_11_0.md:11]: I'm not sure what this means - should 6.0 be changed to 6.2 and 6.2 changed to 6.5? I will leave this for @ssokolovich to update

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Host_Advanced_Hunting.yml (hunk 1 above)
Comment (ssokolovich) [reply Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Host_Advanced_Hunting.yml:218]: Yes, this task name is correct.

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Retrieve_File_README.md (hunk 9 above)
Comment (ssokolovich) [reply Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Retrieve_File_README.md:22]: @David-BMS - No, this PB was intended to retrieve multiple files from multiple endpoints by using looping only the file paths.  Did you change the logic on your end?

Comment (ssokolovich) [reply Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Retrieve_File_README.md:22]: Spoke with @David-BMS , This PB is looping on File Paths and the Endpoint id will be retrieved from the incident field 'Device ID'. 
@julieschwartz18 - I have changed the description a bit - so the users will be aware of this.
`"This playbook uses the Live Response feature to retrieve a file from an endpoint./nNote that the endpoint id will be set from the incident field \"Device ID\"."`

File: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_11_0.md (hunk 12 above)
Comment (ssokolovich) [reply Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_11_0.md:11]: It means that all the new PBs must be 6.5. Checking that.

**Review by David-BMS — APPROVED** [review]

**Pull request merged**